azure-pipelines.yml

```diff
@@ -164,6 +165,7 @@ stages:
           key: 'yarn | "$(osName)" | yarn.lock'
           restoreKeys: |
              yarn | "$(osName)"
+             yarn
```

Comment:
purge this back out, it was removed to ensure no cross OS cache restore for yarn